src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1595,7 +1612,10 @@ namespace UnitarySystems {
             if (!this->m_HeatCoilExists) state.dataSize->ZoneCoolingOnlyFan = true;
             TempSize = this->m_MaxCoolAirVolFlow;
             SaveCurDuctType = state.dataSize->CurDuctType;
-            state.dataSize->CurDuctType = DataHVACGlobals::Cooling;
+            // might want to rethink this method. Tries to find the larger of cooling or heating capcity
+            // however, if there is no heating coil the cooling air flow rate is used, not the main flow rate
+            // this is fine if there are no other systems on the branch. CoilSystem does not do this (#8761).
+            if (this->UnitType == "AirLoopHVAC:UnitarySystem") state.dataSize->CurDuctType = DataHVACGlobals::Cooling;
```

Comment:
UnitarySystem is the only parent that tries to find a seasonal capacity by using the actual cooling or heating flow rate. This hack got diffs back in line. I have a feeling that doing this the way UnitarySystem does could size the system flow rate for cooling only systems different than an external fan on the same branch.